changelog/generated/v1.4.2.md

```diff
@@ -0,0 +1,5 @@
+## v1.4.2
+
+### Bug Fixes
+
+- Bump cryptography dependency in ansible-operator to 3.3.2. This will fix a potential security bug related to update() calls. Ref: https://github.com/pyca/cryptography/blob/main/CHANGELOG.rst#332---2021-02-07. ([#4531](https://github.com/operator-framework/operator-sdk/pull/4531))
```

Comment:
@varshaprasad96 there should be more entries than just this one right? https://github.com/operator-framework/operator-sdk/pull/4534/files#diff-3d4efde7d2c494084cc25d283c8a1901dd42a258759ccb7f1534ad8d9ac35196

Reply:
yeah, missed this!